src/redux/actions/sync.js

```diff
@@ -31,6 +31,36 @@ export function doSetSync(oldHash, newHash, data) {
   };
 }
 
+export function doSetDefaultAccount() {
+  return dispatch => {
+    dispatch({
+      type: ACTIONS.SET_DEFAULT_ACCOUNT,
+    });
+
+    Lbry.account_list().then(accountList => {
+      const { lbc_mainnet: accounts } = accountList;
+      let defaultId;
+      for (let i = 0; i < accounts.length; ++i) {
+        if (accounts[i].satoshis > 0) {
+          defaultId = accounts[i].id;
+          break;
+        }
+      }
+
+      // In a case where there's no balance on either account
+      // assume the second (which is created after sync) as default
+      if (!defaultId && accounts.length > 1) {
+        defaultId = accounts[1].id;
+      }
+
+      // Set the default account
+      if (defaultId) {
```

Comment:
If there is no `defaultId`, should we error to say that sync won't work for whatever reason?

Reply:
No need to throw an error here. If no default id is found, then the wallet will end up being set to the pre-existing default account when it was created.